02-oop-inheritance/topic-2.js

```diff
@@ -0,0 +1,166 @@
+$(document).ready(function(){
+	/*Define Movie*/
+var Movie = function () {
+  /*Set Vars*/
+  this.attributes = {};
```

Comment:
You could directly write lines #5 and #6 into one declaration:
this.attributes = {
  actors: []
};